face.py

```diff
@@ -25,4 +29,4 @@ def mark_faces(image_filename):
     image.save(image_filename.with_suffix(".out.jpg"))
 
 
-mark_faces("Apollo_11_Crew.jpg")
+if __name__ == '__main__': typer.run( mark_faces )
```

Comment:
```suggestion
if __name__ == '__main__':
    typer.run(mark_faces)
```
DevOps Engineer: This should satisfy black.